qemu/tests/rng_bat.py

```diff
@@ -1,13 +1,13 @@
 import re
 import logging
 
-from autotest.client.shared import error
-from autotest.client.shared import utils
-
 from virttest import utils_misc
+from virttest import error_context
```

Comment:
Hi @PandaWei,

1. error module could not get the exception of test case, need to fix it first, otherwise it's difficult to debug the test
https://github.com/avocado-framework/avocado-vt/issues/515

2. it's really not a good workflow to support two framework, 
but it will effect your test, so discuss with @xutian  that we will support both avocado and autotest for the situation (1. comment features, e.g block, nic, storage, image format related 2. old cases), others like virtio-win related cases, we will support avocado only, is ok for you ?

Thanks for your comment